### PR TITLE
Code Review: issue 25: Initial Setup: README, Linter and testing

### PR DESCRIPTION
**Pull request by Ghost---Shadow**
https://github.com/social-lending-mvp-organization/social-credit/issues/25

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    "extends": "airbnb",
+    "env": {
+        "jest": true,
+        "es6": true,
+    }
+};
```
Comment (KshipraShetty) [inline .eslintrc.js]: Add a new line. It's showing "no newline at the end of file"

File: package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "homepage": "https://github.com/social-lending-mvp-organization/social-credit#readme",
+  "devDependencies": {
+    "eslint": "^4.16.0",
+    "eslint-config-airbnb": "^16.1.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-react": "^7.6.1",
+    "jest": "^22.1.4"
+  }
+}
```
Comment (Nidhi1461996) [inline package.json]: New line to be added at the end of the file

File: package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "coverage": "jest --coverage"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TechUniv2018/social-credit.git"
+  },
+  "author": "group3",
```
Comment (saurav-sahu) [inline package.json]: Use the 'contributors' property to add all group members' names.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/social-lending-mvp-organization/social-credit/issues"
```
Comment (saurav-sahu) [inline package.json]: Update bugs URL to TechUniv2018 organization.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "homepage": "https://github.com/social-lending-mvp-organization/social-credit#readme",
```
Comment (saurav-sahu) [inline package.json]: Update homepage URL to TechUniv2018 orgranization.

File: package.json (hunk 2 above)
Comment (Ghost---Shadow) [reply package.json]: This is a json file, not js. The eslint removes new line.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  "scripts": {
+    "test": "jest",
+    "coverage": "jest --coverage"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/TechUniv2018/social-credit.git"
+  },
+  "contributors": [
+    "Kshirpa Shetty <kshiprashetty@gmail.com>",
```
Comment (saurav-sahu) [inline package.json]: Fix spelling. (Kshipra)

**Review by mrsauravsahu — APPROVED** [review]

**Pull request merged**